site/styles/_carousel.scss

```diff
@@ -226,17 +223,45 @@ s72-carousel {
       margin-left: 0.5rem;
       margin-right: 0.5rem;
     }
-
+    s72-pricing-buttons .s72-btn {
+      margin-bottom: 10px;
+    }
     // Handles spacing when there is only rent, only buy, or only play button
-    s72-pricing-buttons .s72-btn,
     s72-play-button {
-      &:nth-child(1) {
+      &:nth-child(1),
+      &:nth-child(2) {
         margin-right: 0;
       }
       &:only-child {
         margin-right: 0.5rem;
       }
     }
+
+    .s72-pricing-button-container .s72-btn:nth-child(2) {
+      margin-left: 0; //override relish
```

Comment:
Interesting comment, perhaps Relish shouldn't be setting styles.

Reply:
I wonder if we should remove the Relish CSS link from the `<head>` ?

Reply:
Does this need a wider discussion?
cc @additiverich  

Also does this hold up a merge if everything else is okay?

Reply:
IMO it's not something to worry about in this PR... just something to think about.